.github/workflows/doc.yml

```
@@ -32,6 +33,37 @@ jobs:
     - uses: actions/checkout@v3
     - name: Register Sphinx problem matcher
       run: echo "::add-matcher::.github/problem-matchers/sphinx.json"
+    - name: 'Setup Python'
```

"Setup" is a noun, this is with verb:
```suggestion
    - name: 'Set up Python'
```